src/distilabel/utils/card/distilabel_template.md

```diff
@@ -70,6 +70,21 @@ ds = load_dataset("{{ repo_id }}")
 </details>
 {% endfor %}
 
+{% if artifacts %}
+## Artifacts
+
+{% for step_name, artifacts in artifacts.items() %}
+* **Step**: `{{ step_name }}`
+  {% for artifact in artifacts %}
+    * **Artifact name**: `{{ artifact.name }}`
+      {% for name, value in artifact.metadata.items() %}
+        * `{{ name }}`: {{ value }}
```

Comment:
The `value` of the artifact's metadata here points to the object stored in the repo? I'm not sure, maybe I missed in the code, but would be helpful to easily point to it

Reply:
Renders the values in the `metadata.json` of each artefact, is that what you were asking?